lib/pierre.py

```diff
@@ -28,8 +28,11 @@ def check(payload, headers, host):
         return reply
 
     owner, repo = get_owner_and_repo(payload)
+    dependency_id = get_dependency_id(payload)
```

Comment:
rename `dependency_id` to `this_pr_id` or something that explains that we want this PRs number/id